go/libraries/doltcore/sqle/logictest/DoltReleaseBenchmarkJenkinsfile

```diff
@@ -0,0 +1,89 @@
+                    agent {
+                        kubernetes {
+                            label "liquidata-inc-ld-build"
+                        }
+                    }
+                    environment {
+                        PATH = "${pwd()}/.ci_bin:${pwd()}/.ci_bin/node_modules/.bin:${env.PATH}"
+                        TMPDIR = "${pwd()}/tempDir"
+                        CREDSDIR = "/home/jenkins/.dolt/creds"
+                        DOLT_CREDS = credentials("system-account-dolthub-creds")
+                        DOLT_GLOBAL_CONFIG = credentials("system-account-dolthub-config")
+                        DOLT_CONFIG_PATH = "/home/jenkins/.dolt"
+                    }
+                    steps {
+                        dir ("sqllogictest") {
+                            git url: "https://github.com/liquidata-inc/sqllogictest.git"
+                        }
+                        dir ("dolt-release") {
+                            checkout([$class: 'GitSCM', branches: [[name: "${DOLT_RELEASE}"]],
+                                 userRemoteConfigs: [[url: 'https://github.com/liquidata-inc/dolt.git']]])
+                        }
+                        dir ("dolt-release/go") {
+                            sh "rm -rf $TMPDIR && mkdir $TMPDIR"
+                            sh "go get -mod=readonly ./..."
+                            sh "go build -mod=readonly -o ../../.ci_bin/dolt ./cmd/dolt/."
+                        }
+                        dir ("go/libraries/doltcore/sqle/logictest/main") {
+                            sh "rm -rf $DOLT_CONFIG_PATH && mkdir -p $DOLT_CONFIG_PATH"
+                            sh "echo '$DOLT_GLOBAL_CONFIG' > $DOLT_CONFIG_PATH/config_global.json"
+                            sh "rm -rf $CREDSDIR && mkdir $CREDSDIR"
+                            sh "cat $DOLT_CREDS > /home/jenkins/.dolt/creds/${CREDS_HASH}.jwk"
+                            sh "dolt config --global --add user.creds $CREDS_HASH"
+                            sh "dolt config --global --add metrics.disabled true"
+                            sh "rm -rf temp && mkdir temp"
+                            script {
+                                def j = env.TEST_N_TIMES.toInteger()
+                                for (int i = 1; i <= j; ++i) {
+                                    sh "rm -rf .dolt && dolt init"
+                                    sh "go run . run ../../../../../../sqllogictest/test/select1.test > temp/results${i}.log"
+                                    sh "go run . parse \"$DOLT_RELEASE\" temp/results${i}.log > temp/parsed${i}.json"
+                                }
+                                sh "rm -rf .dolt"
+                            }
+                        }
+                        dir ("go") {
+                            sh "go get -mod=readonly ./..."
+                            sh "go build -mod=readonly -o ../.ci_bin/dolt ./cmd/dolt/."
+                        }
```

Comment:
After we test dolt at a particular release, we rebuild it from the current checkout (`master`) so we have access to all it's features below.